.devcontainer/awscli-install-run-once.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+apt install unzip
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+./aws/install
```

Comment:
```suggestion
./aws/install

```

Reply:
Fixed in 4490493cac43fe134f839c44b25291c96a687d51